packages/discord.js/src/structures/ThreadMember.js

```diff
@@ -35,12 +35,22 @@ class ThreadMember extends Base {
      */
     this.id = data.user_id;
 
-    this._patch(data);
+    this._patch(data, extra);
   }
 
-  _patch(data) {
+  _patch(data, extra = {}) {
     if ('join_timestamp' in data) this.joinedTimestamp = Date.parse(data.join_timestamp);
     if ('flags' in data) this.flags = new ThreadMemberFlagsBitField(data.flags).freeze();
+
+    /**
+     * The guild member associated with this thread member.
+     * @type {?GuildMember}
+     */
+    if ('member' in data) {
```

Comment:
Are the docs still recognised if they are above `if`?

```suggestion
    if ('member' in data) {
      /**
       * The guild member associated with this thread member.
       * @type {?GuildMember}
       */
```